src/packagedcode/pypi.py

```diff
@@ -303,6 +305,33 @@ def parse_setup_py(location):
     )
 
 
+def parse_requirements_txt(location):
+    """
+    Return a Package built from requirements.txt.
+    """
+    dependencies = OrderedDict()
+    with io.open(location, encoding='utf-8') as loc:
+        for req in requirements.parse(loc):
+            dependencies[req.name] = req.specs
+    package_dependencies = []
+    for dep_name, dep_specs in dependencies.items():
+        package_dependencies.append(
+            models.DependentPackage(
+                purl=PackageURL(
+                    type='pypi',
+                    name=dep_name
+                ).to_string(),
+                requirement=''.join(dep_specs[0]),
+                scope='dependencies',
```

Comment:
Could there be a smarter way to infer a scope?
- Plain `requirements.txt` files would be `runtime` IMHO or whatever scope we use in for setup.py `install_requires`
-  `requirements_test.txt` or `requirements-tests.txt`  etc.. --> `test` scope?
-  `requirements_dev.txt` or `requirements-dev.txt`  etc.. --> `dev` scope?
...
etc?